server/daos/authors.js

```diff
@@ -2,5 +2,26 @@ import db from '@database/models';
 
 export const insertAuthor = args => db.books.create(args);
 
-export const updateAuthor = async ({ id, name, country, age }) =>
-  db.authors.update({ name, country, age }, { where: { id } });
+export const updateAuthor = async ({ id, name, country, age }, { fetchUpdated }) => {
+  const options = {};
+
+  options.where = { id };
+
+  if (fetchUpdated) {
+    options.returning = true;
+  }
+
+  const authorRes = await db.authors.update({ name, country, age }, options);
+
+  let affectedRowData;
+
+  if (fetchUpdated) {
+    // authorRes = returns an array with one or two elements. The first element is always the number of affected rows, while the second element is the actual affected rows(PostgresSQl only)
+    const affectedRow = authorRes[1];
+
+    // affectedRow = gets an array of all affected row elements(in our case we will always get a single element)
+    affectedRowData = affectedRow[0];
+  }
```

Comment:
this is more of a code style thing. 

typically what I prefer to make it more readable is this
```
if (!fetchUpdated) {
  return authorRes
}

// continue for code with when `fetchUpdated = true`